sdcm/tester.py

```diff
@@ -817,10 +832,10 @@ def is_encrypt_keys_needed(self):
 
     def prepare_kms_host(self) -> None:
         if (self.params.is_enterprise and ComparableScyllaVersion(self.params.scylla_version) >= '2023.1.3'
-            and self.params.get('cluster_backend') == 'aws'
-            and not self.params.get('scylla_encryption_options')
-            and self.params.get("db_type") != "mixed_scylla"  # oracle probably doesn't support KMS
-            ):
+                and self.params.get('cluster_backend') == 'aws'
```

Comment:
pycharm keep insisting on this one....

Reply:
it is changing by pylint. It is changed every time when I run pylint

Reply:
Use `git add -p %file%` approach to commit only part of the changes in a file.

Reply:
I'd just merge it so we never see it again - no need to waste time on thinking about it.